test/test_migrate.rb

```diff
@@ -35,33 +54,29 @@
     end
     
     test 'repair broken links' do
-      PREFER_RELATIVE = true
-      RUN_SILENT = true
-      NO_DRY_RUN = true
-      HYPHENATE = false
-      
-      Dir.chdir(@path) do
-        load script_path
-      end
+      load_script
+    
       f = ::File.new(::File.join(@path, 'Subdir/Foo.md'), 'r')
       assert_equal result, f.read
     end
     
-    test 'change spaced filenames to hyphenated filenames' do
-      RUN_SILENT = true
-      NO_DRY_RUN = true
-      PREFER_RELATIVE = true
-      HYPHENATE = true
-    
-      Dir.chdir(@path) do
-        load script_path
-      end
+    test 'change spaced filenames to hyphenated filenames' do    
+      load_script(hyphenate: true)
     
       f = ::File.new(::File.join(@path, 'Home.textile'), 'r')
       output = f.read
       assert_equal true, output.include?('[[Bilbo-Baggins.md]]')
       assert_equal true, output.include?('[[evil|Mordor/Eye-Of-Sauron.md]]')
     end
+    
+    test 'migration with page file dir' do      
```

Comment:
Do you test `--write` anywhere?

Reply:
`NO_DRY_RUN` is true for all the tests! See line 29

Reply:
Perfect. 👍